lib/mongodb.js

```diff
@@ -39,6 +40,7 @@ function ObjectID(id) {
   }
 }
 
+exports.generateMongoDBURL = generateMongoDBURL;
```

Comment:
That would only work if we were injecting our new class somehow. Our line 74 looks like `  dataSource.connector = new MongoDbVersioned(s, dataSource);`. If we want to remove those two exports then the best way would probably be to simply make them properties of the MongoDB function so they are available to subclasses.